### DISCUSSION TITLE
OC 0.9.8 + 2017 MacBook Pro SMBIOS

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [MultimediaLucario/Lenovo-ThinkPad-T440s#38] [Announcements]
![repository-open-graph-t440s](https://github.com/MultimediaLucario/Lenovo-ThinkPad-T440s/assets/72415505/f4225004-f1c4-4bab-a97d-4223294433b5)

# First Update since 2023! 🎉

This is the first update in a long time. However, I still do not have a T440s. So, feel free to test this yourselves and send any feedback my way. All of the EFI folders are pretty self-explanatory.

**Full Changelog**: https://github.com/MultimediaLucario/Lenovo-ThinkPad-T440s/compare/OC-0.8.8...OC-0.9.8

<hr /><em>This discussion was created from the release <a href='https://github.com/MultimediaLucario/Lenovo-ThinkPad-T440s/releases/tag/OC-0.9.8'>OC 0.9.8 + 2017 MacBook Pro SMBIOS</a>.</em>

**Comment by TeamBattino** [comment]
I dont quite understand. For instance if i want to install montrey on my T440s Thinkpad, do I need the Monterey EFI, the T440s EFI or a combination of the two?

**Comment by QuacksCHamois** [comment]
Hi, I tried this update, and the one  from March on my T440S using Monterey.  I tried both Monterey and Ventura, each create a boot loop cycle and force restart until I went back to 2023 's rock solid EFI.  Any advice?  Thanks!

**Comment by luisangeldom** [comment]
hey i was trying your guide and on step 7 on the installation part it says this
"Now after install again boot into your usb drive and then select the drive that you installed macOS on."
but whne i boot it again with the usb connected it doesnt appear in the booting drives and i have it on top on my boot sequence, i tried with other usbs because i thought maybe it was corrupted but nothing worked, could you make a video tutorial for that please, i think i am lost in one of the steps that you dont explain, i am new to hackintosh but maybe it woudl be good to have  aguide in video  as well
thanks in advace, greetings
